en/asgardeo/docs/apis/restapis/functionality.yaml

```diff
@@ -0,0 +1,346 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'PUT' \
+            'https://api.asgardeo.io/t/{organization-name}/api/users/v1/{user-id}/user-functionality/{function-id}' \
+            -H 'accept: */*' \
+            -H 'Authorization: Bearer {bearer_token}' \
+            -H 'Content-Type: application/json' \
+            -d '{
+            "action": "LOCK",
+            "timeToLock": 1594987178,
+            "functionalityLockReason": "SecurityQuestionBasedBased",
+            "functionalityLockReasonCode": "MaxFailedAttempts"
+            }'
+components:
+  schemas:
+    StatusChangeRequest:
+      type: object
+      required:
+        - action
+      properties:
+        action:
+          type: string
+          enum: [ LOCK , UNLOCK]
+          example: lock
+          description: LOCK/UNLOCK
+        timeToLock:
+          type: string
+          example: 1594987178
+          description: Time in millisecond
+        functionalityLockReason:
+          type: string
+          example: SecurityQuestionBasedBased
+        functionalityLockReasonCode:
+          type: string
+    LockStatusResponse:
+      type: object
+      properties:
+        lockStatus:
+          type: boolean
+          example: true
+          description: returns true if it is locked
+        unlockTime:
+          type: string
+          example: SecurityQuestionBasedBased
+        lockReasonCode:
+          type: string
+        lockReason:
+          type: string
+          example: SecurityQuestionBasedBased
+    UserStatusChangeRequest:
+      type: object
+      required:
+        - action
+      properties:
+        action:
+          type: string
+          enum: [ LOCK , UNLOCK]
+          example: lock
+          description: LOCK/UNLOCK
+        timeToLock:
+          type: string
+          example: 1594987178
+          description: Time in millisecond
+
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: "AAA-00000"
+        message:
+          type: string
+          example: "Some Error Message"
+        description:
+          type: string
+          example: "Some Error Description"
+        traceId:
+          type: string
+          example: "e0fbcfeb-3617-43c4-8dd0-7b7d38e13047"
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://api.asgardeo.io/t/{organization-name}/oauth2/authorize'
+          tokenUrl: 'http://api.asgardeo.io/t/{organization-name}/oauth2/token'
+          scopes: {}
+servers:
+  - url: 'https://api.asgardeo.io/t/{organization-name}/api/users/v1'
```

Comment:
missing new line at the eof

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/08dc04ec98014b3151364f00d94ecbdc3880d7f1